src/index.html

```diff
@@ -11,7 +11,178 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="personal_information"
+      autocomplete="off">
```

Comment:
check the closing tag
![image](https://user-images.githubusercontent.com/22824431/230053417-939910f1-eb28-4d50-87ab-ab2549e5febb.png)
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
      name="personal_information"
      autocomplete="off"
    >
```